static/js/app.js

```diff
@@ -1,5 +1,5 @@
 function dismissNotification() {
-	const notification = document.querySelector(".js-notification");
+	const notification = document.querySelector(".messages");
```

Comment:
This closes the entire wrapper, eliminating any remaining padding/margin space.

Reply:
I would recommend moving the `.js-notification` to keep the class name following the [naming convention docs](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md#class-naming-convention).